webservices/templates/swagger-ui.html

```diff
@@ -23,9 +23,45 @@
 
       body
       {
-        margin:0;
-        background: #fafafa;
+      margin:0;
+      background: #fafafa;
       }
+      
+      label[for=user_first_name]:before
+      {
+      content: " *";
+      border-style: none;
+      cursor: default;
+      color: #E87723;
```

Comment:
I don't think you  need `border-style` or `cursor`  for any of the `:before`s , unless you put them in for s specific reason I am missing

Reply:
fixed!  Thanks @johnnyporkchops